R/get_analysis_date.R

```diff
@@ -35,7 +35,7 @@
 #' where date1, date2, date3, date4, date5 represent the analysis dates determined
 #' solely by Condition 1, Condition 2, Condition 3, Condition 4 and Condition 5,
 #' respectively.
-#'
+#' }
```

Comment:
This closing bracket should be after the final `\item{}{}` line. Only text within `\item{}{}` should be contained with the `\describe{}`

Reply:
Good point